client/components/deposit-status-chip/test/index.test.tsx

```diff
@@ -10,13 +10,6 @@ import { render } from '@testing-library/react';
 import DepositStatusChip from '..';
 
 describe( 'Deposits status chip renders', () => {
-	test( 'Renders In Transit status chip.', () => {
-		const { getByText } = render(
-			<DepositStatusChip status="estimated" />
-		);
-		expect( getByText( 'Estimated' ) ).toBeTruthy();
-	} );
-
 	test( 'Renders In Transit status chip.', () => {
```

Comment:
Let's give these more accurate test names while we're here, I'm assuming each one tests a specific status. E.g. `Renders pending status chip` (or whatever it really means to merchants).

Reply:
Great catch: updated in bea8783.